src/cpp/multisolver_interface/CMakeLists.txt

```diff
@@ -68,5 +73,6 @@ if(COIN_OR)
 			Coin::CoinUtils
 			Coin::Osi
 			Coin::Cbc
+		   ${CMAKE_DL_LIBS}
```

Comment:
I recommend creating a library to load XPRESS, and to link it against `-ldl`

Also, `-ldl` doesn't exist in Windows